12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,470 @@
+
+### Concepts
+
+<details>
+
+<summary>Content item JSON structure</summary>
+
+The Delivery API outputs the JSON structure outlined below to represent the retrieved content items, which consist of a range of properties:&#x20;
+
+* Basic properties for any content item include `name`, `id` and `contentType`.
+* All editorial properties from the content type can be found in the `properties` collection. Depending on the configured property editor, the property output value can be a _string_, _number_, _boolean expression_, _array_, _object_ or _`null`_.
+* The `route` property provides the `path` of the content item, as well as details about the root node value that is represented by the `startItem` object. We will discuss the concept of a `startItem` in more detail in the next section.
+* If the content item varies by culture, the `cultures` property will contain information about all configured cultures for the content node, including the culture variant `path` and `startItem` for each one.
+
+```json
+{
+  "name": "string",
+  "route": {
+    "path": "string",
+    "startItem": {
+      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "path": "string"
+    }
+  },
+  "id": "11fb598b-5c51-4d1a-8f2e-0c7594361d15",
+  "contentType": "string",
+  "properties": {
+    "property1Alias": "string",
+    "property2Alias": 0,
+    "property3Alias": true,
+    "property4Alias": [],
+    "property5Alias": {},
+    "property6Alias": null
+  },
+  "cultures": {
+    "cultureIsoCode1": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    },
+    "cultureIsoCode2": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    }
+  }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Start item</summary>
+
+When working headlessly in a multi-site setup, sometimes it can be difficult to determine the site context of a particular content item. This is where the _start item_ comes into play.
+
+The start item represents the root of a content item and is commonly returned from the API in conjunction with a content path. For instance:
+
+```json
+{
+  "route": {
+    "path": "/articles/2023/getting-started",
+    "startItem": {
+      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "path": "docs-portal"
+    }
+  }
+}
+```
+
+This means that the content item resides under the root node with "docs-portal" as an alias and can be retrieved using the path "/articles/2023/getting-started".
+
+
+
+The start item is also quite helpful when obtaining content from the API. By supplying either the start item `id` or `path` in the `Start-Item` header, you can instruct the Delivery API to use the corresponding root node as the starting point for the requested content operation:
```

Comment:
I tried to shorten the sentence. Didn't succeed. Can you give it a try? Perhaps it could be split into two?

```suggestion
The start item is helpful when obtaining content from the API. Supplying the start item `id` or `path` in the `Start-Item` header, allows you to instruct the Delivery API to use the corresponding root node as the starting point:
```

Reply:
Maybe something like this:
> The start item can also be quite helpful through the `Start-Item` request header when obtaining content from the Delivery API. Supplying a root node `id` or `path` as the header value, allows you to specify which is the starting point for the requested content operation:

OR 

> The start item is also quite helpful when obtaining content from the Delivery API by applying the `Start-Item` request header. Supplying a root node `id` or `path` as the header value, allows you to specify which is the starting point for the requested content operation: